Source/SpatialGDK/Private/SpatialInteropPipelineBlock.cpp

```diff
@@ -353,9 +359,13 @@ void USpatialInteropPipelineBlock::CreateActor(TSharedPtr<worker::Connection> Lo
 
 	if (EntityActor)
 	{
-		UClass* ActorClass = GetNativeEntityClass(MetadataComponent);
-		USpatialInterop* Interop = NetDriver->GetSpatialInterop();
-		check(Interop);
+		// To account for stably named actors
```

Comment:
This comment doesn't really help explain what's going on here.  Why are we removing stable named objects.

Reply:
Added more explanation, but I think we also discussed that this function shouldn't be called RemoveStablyNamedObject, but RemoveStablyNamedObjectReference/Mapping/etc.